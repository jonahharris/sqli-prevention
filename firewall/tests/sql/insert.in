>>> Sada testovacich SQL dotazu: insert
INSERT INTO a(b, c) VALUES(1 ,2);
INSERT INTO t1 VALUES (10,10), (20,20);
INSERT LOW_PRIORITY INTO a(b, c) VALUES(1 ,2);
INSERT DELAYED INTO a(b, c) VALUES(1 ,2);
INSERT HIGH_PRIORITY INTO a(b, c) VALUES(1 ,2);
INSERT LOW_PRIORITY IGNORE INTO a(b, c) VALUES(1 ,2);
INSERT DELAYED IGNORE INTO a(b, c) VALUES(1 ,2);
INSERT HIGH_PRIORITY IGNORE INTO a(b, c) VALUES(1 ,2);
INSERT IGNORE INTO a(b, c) VALUES(1 ,2);
INSERT INTO person (first_name, last_name) VALUES ('John', 'Doe');
INSERT INTO tbl_name VALUES (1, "row 1"), (2, "row 2");
INSERT INTO person SET first_name = 'John', last_name = 'Doe';
INSERT INTO contractor SELECT * FROM person WHERE status = 'c';
INSERT INTO a (b, c) VALUES (1,"2004-08-09", 15) ON DUPLICATE KEY UPDATE c = c + 15;
INSERT INTO person SET first_name = DEFAULT;
INSERT INTO person SET person.first_name = DEFAULT;
INSERT INTO person SET first_name = DEFAULT, last_name = DEFAULT;
INSERT INTO person SET person.first_name = DEFAULT, person.last_name = DEFAULT;
