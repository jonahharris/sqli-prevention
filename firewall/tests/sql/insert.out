>>> Sada testovacich SQL dotazu: insert
COLUMN b;COLUMN c;INSERTCOLS 2;NUMBER 1;NUMBER 2;VALUES 2;TABLE a;INSERT VALS 0 1;STMT;1;0;
NUMBER 10;NUMBER 10;VALUES 2;NUMBER 20;NUMBER 20;VALUES 2;TABLE t1;INSERT VALS 0 2;STMT;1;0;
COLUMN b;COLUMN c;INSERTCOLS 2;NUMBER 1;NUMBER 2;VALUES 2;TABLE a;INSERT VALS 1 1;STMT;1;0;
COLUMN b;COLUMN c;INSERTCOLS 2;NUMBER 1;NUMBER 2;VALUES 2;TABLE a;INSERT VALS 2 1;STMT;1;0;
COLUMN b;COLUMN c;INSERTCOLS 2;NUMBER 1;NUMBER 2;VALUES 2;TABLE a;INSERT VALS 4 1;STMT;1;0;
COLUMN b;COLUMN c;INSERTCOLS 2;NUMBER 1;NUMBER 2;VALUES 2;TABLE a;INSERT VALS 9 1;STMT;1;0;
COLUMN b;COLUMN c;INSERTCOLS 2;NUMBER 1;NUMBER 2;VALUES 2;TABLE a;INSERT VALS 10 1;STMT;1;0;
COLUMN b;COLUMN c;INSERTCOLS 2;NUMBER 1;NUMBER 2;VALUES 2;TABLE a;INSERT VALS 12 1;STMT;1;0;
COLUMN b;COLUMN c;INSERTCOLS 2;NUMBER 1;NUMBER 2;VALUES 2;TABLE a;INSERT VALS 8 1;STMT;1;0;
COLUMN first_name;COLUMN last_name;INSERTCOLS 2;STRING '***';STRING '***';VALUES 2;TABLE person;INSERT VALS 0 1;STMT;1;0;
NUMBER 1;STRING '***';VALUES 2;NUMBER 2;STRING '***';VALUES 2;TABLE tbl_name;INSERT VALS 0 2;STMT;1;0;
STRING '***';ASSIGN first_name;STRING '***';ASSIGN last_name;TABLE person;INSERT ASGN 0 2;STMT;1;0;
SELECTALL;TABLE person;NAME status;STRING '***';CMP 4;WHERE;SELECT 0 1 1;TABLE contractor;INSERT SELECT 0;STMT;1;0;
COLUMN b;COLUMN c;INSERTCOLS 2;NUMBER 1;STRING '***';NUMBER 15;VALUES 3;NAME c;NUMBER 15;ADD;ASSIGN c;DUPUPDATE 1;TABLE a;INSERT VALS 0 1;STMT;1;0;
DEFAULT;ASSIGN first_name;TABLE person;INSERT ASGN 0 1;STMT;1;0;
DEFAULT;ASSIGN person.first_name;TABLE person;INSERT ASGN 0 1;STMT;1;0;
DEFAULT;ASSIGN first_name;DEFAULT;ASSIGN last_name;TABLE person;INSERT ASGN 0 2;STMT;1;0;
DEFAULT;ASSIGN person.first_name;DEFAULT;ASSIGN person.last_name;TABLE person;INSERT ASGN 0 2;STMT;1;0;
